### DISCUSSION TITLE
how safe are the automated backups

### DISCUSSION THREAD

**Discussion by orenaksakal** [pocketbase/pocketbase#5063] [Q&A] [answered]
I wonder how safe are the automated backups at scale? Let's say my db exceeds 50gb would it still be able to backup the db successfully to R2? Should I have another automated rsync/backup solution instead?

**Comment by ganigeorgiev** [comment, marked as answer]
The automated backups are full snapshot of your `pb_data` directory and are not suitable for large scale datasets (eg. 5GB+).

If you are operating with 50GB+ you'll be better with some other backup mechanism, ideally that supports incremental/partial backups.
